vscode/package.json

```diff
@@ -101,7 +101,8 @@
           "default": "unrestricted",
           "enum": [
             "unrestricted",
-            "base"
+            "base",
+            "adaptive"
           ],
           "enumDescriptions": [
```

Comment:
do you need to add an enum description for adaptive here, even just a placeholder?